lib/database/postgresql.go

```diff
@@ -0,0 +1,83 @@
+package database
+
+import (
+	"context"
+	"fmt"
+
+	_ "github.com/jackc/pgx" // PostgrerSQL driver
+	"github.com/jackc/pgx/v4"
+	"github.com/sdslabs/gasper/configs"
+	"github.com/sdslabs/gasper/types"
+)
+
+var (
+	postgresqlHost         = `%`
+	postgresqlRootUser     = configs.ServiceConfig.Kaen.PostgreSQL.Env["POSTGRES_USER"]
+	postgresqlPassword     = configs.ServiceConfig.Kaen.PostgreSQL.Env["POSTGRES_PASSWORD"]
+	postgresqlDatabaseName = configs.ServiceConfig.Kaen.PostgreSQL.Env["POSTGRES_DB"]
+	postgresqlPort         = configs.ServiceConfig.Kaen.PostgreSQL.ContainerPort
+)
+
+// CreatePostgresqlDB creates a postgre database
+func CreatePostgresqlDB(db types.Database) error {
+	ctx := context.Background()
+	connection := fmt.Sprintf("postgres://%v:%v@localhost:%d/%v", postgresqlRootUser, postgresqlPassword, postgresqlPort, postgresqlRootUser)
+	conn, err := pgx.Connect(ctx, connection)
+	if err != nil {
+		return fmt.Errorf("Error while creating the database : %s", err)
+	}
+	defer conn.Close(ctx)
+
+	if _, err = conn.Exec(ctx, "CREATE DATABASE "+db.GetName()); err != nil {
```

Comment:
Could be made simpler with another sprintf() (as done everywhere else)

Reply:
Using sprintf() would be much better to avoid bugs caused by things like:
```go
conn.Exec(ctx, "CREATE DATABASE"+db.GetName()); // Here the query string will be "CREATE DATABASEMyDataBase" instead of "CREATE DATABASE MyDataBase"
```